servers/jax-rs-tests/src/test/java/org/projectnessie/jaxrs/AbstractTestJerseyRest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.jaxrs;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.projectnessie.jaxrs.ext.NessieJaxRsExtension;
+import org.projectnessie.versioned.persist.adapter.DatabaseAdapter;
+import org.projectnessie.versioned.persist.tests.extension.DatabaseAdapterExtension;
+import org.projectnessie.versioned.persist.tests.extension.NessieDbAdapter;
+
+@ExtendWith(DatabaseAdapterExtension.class)
+abstract class AbstractTestJerseyRest extends AbstractTestRest {
```

Comment:
what's the difference now between this class and `TestJerseyRest`? I find it a bit confusing that there's a `AbstractTestJerseyRest` that's primarily there to support the new DB adapters and stuff and that there's the existing test classes. I think we should make it clear in terms of naming what's what. wdyt?

Reply:
The old ones are going to be entirely removed and once that's done, the hierarchy can be flattened/simplified.